src/openms/source/FORMAT/XTandemInfile.cpp

```diff
@@ -685,11 +692,21 @@ namespace OpenMS
     return refine_;
   }
 
+  bool XTandemInfile::isNoiseSuppression() const
```

Comment:
the function name is weird.
I can see that it nicely fits to isRefining(), but useNoiseSuppression
or getNoiseSuppression would be preferred.
Minor note:
In fact, naming all functions getXY() and setXY() make it easier for developers who use and IDE with auto-completion.